File: ProAgil.Domain/Lot.cs
using System;

namespace ProAgil.Domain
{
    public class Lot
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public decimal price { get; set; }
        public DateTime? InitialDate { get; set; }
        public DateTime? FinalDate { get; set; }
        public int amount { get; set; }
        public int EventId { get; set; }
        public Event Event { get; set; }
    }
}

File: ProAgil-App/src/app/models/Lot.ts
import { BaseResourceModel } from './base-resource.model';

export class Lot extends BaseResourceModel {
  constructor(
    id: number,
    name: string,
    price: number,
    initialDate?: Date,
    finalDate?: Date,
    amount?: number,
    eventId?: number
  ) {
    super();
  }
}
